web/themes/joinup/templates/rdf_entity/rdf-entity--asset-distribution--listing-item.html.twig

```diff
@@ -4,21 +4,34 @@
  * Themeing for rdf entities.
  */
 #}
-<div class="listing__card listing__card--detail-container mdl-card">
+{%
+  set classes = [
+    'listing__card',
+    'mdl-card',
+    'listing__card--detail-container',
+  ]
+%}
+{# @todo Implement changes - https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-3883 #}
+<div {{ attributes.addClass(classes) }}>
   <div class="listing__metadata">
     <div class="listing__field">
       <h2 class="listing__title">{{ label }}</h2>
     </div>
     <div class="listing__field">
-      <div class="label gray">{{ 'Weight'|t }}:</div>
-      <div class="value bold">{{ content.field_ad_file_size }}</div>
+      {% if content.field_ad_file_size|render %}
+        <div class="listing__weight listing__single">
+            <div class="label gray">{{ 'Weight'|t }}:</div>
+            <div class="value bold">{{ content.field_ad_file_size }}</div>
+        </div>
+      {% endif %}
+      {% if content.field_ad_licence|render %}
+        <div class="listing__licence listing__single">
+          <div class="label gray">{{ 'Licence'|t }}:</div>
+          <div title="{{ content.field_ad_licence|render|striptags|trim }}" class="value bold underline">{{ content.field_ad_licence }}</div>
```

Comment:
This set of classes `value bold underline` is no BEM by any way. Please try to refactor at least the `value` which is way too generic.

Reply:
I agree with you that these classes are not perfect, but we should leave it as it is, because they are used across the project. Alteration in one place will not change much, but it might creates confusion. Let's create follow up ticket to sort the generic classes and replace them with BEM like. In my opinion it's better to improve this once and for all.